src/scss/components/_login-button-logo.scss

```diff
@@ -33,7 +30,7 @@
   }
 
   &__image {
-    max-height: $grid-row-height;
-    max-width: $grid-row-height;
+    max-height: var(--of-login-button-logo-image-max-max-height, 46px);
```

Comment:
Where is this coming from?

Reply:
sass-variables are static and set at build-time, which is not very flexible. So I stumbled on it and decided to change it into a CSS variable which can be overridden in custom themes!